src/screens/Login/index.test.tsx

```diff
@@ -1,17 +1,48 @@
 import React from 'react';
 
-import { render, screen } from '@testing-library/react';
+import { render, screen, waitFor } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 
+import { formTestIds } from 'components/Form';
 import { authLayoutTestIds } from 'components/Layout/Auth';
+import { fillInput } from 'tests/helpers';
 
-import LoginScreen from '.';
+import LoginScreen, { loginScreenTestIds } from '.';
 
 describe('LoginScreen', () => {
   it('renders the AuthLayout with the correct title', () => {
     render(<LoginScreen />);
 
     const authLayoutHeaderTitle = screen.getByTestId(authLayoutTestIds.headerTitle);
 
-    expect(authLayoutHeaderTitle).toHaveTextContent('auth:sign_in');
+    expect(authLayoutHeaderTitle).toHaveTextContent('auth:heading.sign_in');
+  });
+
+  describe('given the invalid inputs', () => {
```

Comment:
Should we add the valid case to just check that all error messages disappear?

```
describe('given the valid inputs', () => { ... }
```

Reply:
Updated on https://github.com/rosle/react-nimble-survey/pull/25/commits/503ef97d8cae65cd2211e06f619e69390630ac94 👍